package/boot/uboot-sunxi/Makefile

```diff
@@ -270,6 +270,15 @@ define U-Boot/nanopi_neo2
   ATF:=a64
 endef
 
+define U-Boot/nanopi_r1s_h5
+  BUILD_SUBTARGET:=cortexa53
+  NAME:=U-Boot for NanoPi R1S (H5)
```

Comment:
Please remove the prefix `U-Boot for` here.

Reply:
done